### PR TITLE
clients/stellarcore: Support more parameters from the upgrades endpoint

### PR DESCRIPTION
**Pull request by 2opremio**
See https://github.com/stellar/stellar-core/blob/0241e79f74dc017f20e190abd3825873222c5ca5/src/main/CommandHandler.cpp#L568 for alll the supported parameters.

### CODE REVIEW

File: clients/stellarcore/client.go [hunk 1]
```diff
@@ -56,12 +58,105 @@ func drainReponse(hresp *http.Response, close bool, err *error) (outerror error)
...
+	}
+	cmd := exec.Command(corePath, "get-settings-upgrade-txs",
+		config.SigningKey.Address(),
+		strconv.FormatUint(uint64(config.BaseSeqNum), 10),
+		config.NetworkPassphrase,
+		"--xdr", upgradeConfigB64,
+		"--signtxs")
+	inputStr := config.SigningKey.Seed()
+	cmd.Stdin = strings.NewReader(inputStr)
+	out, err := cmd.CombinedOutput()
```
[the remark this answers is not in the document]
Comment (2opremio) [reply clients/stellarcore/client.go:112]: Makes sense. I will fix it

[the remark this answers is not in the document]
Comment (2opremio) [reply clients/stellarcore/client.go:112]: Looking at the code, it seems that `Output` also captures the sdterr and appends it any error it finds. So, simply replacing it was good enough.

File: clients/stellarcore/client.go [hunk 2]
```diff
@@ -56,12 +58,105 @@ func drainReponse(hresp *http.Response, close bool, err *error) (outerror error)
...
+	queryParams := url.Values{}
+	queryParams.Add("configupgradesetkey", keyB64)
+	return c.setUpgradesAt(ctx, at, queryParams)
+}
+
+type GenSorobanConfig struct {
+	BaseSeqNum        uint32
+	NetworkPassphrase string
+	// TODO: Should we ask core to sign the tx or should we do it ourselves?
+	SigningKey *keypair.Full
```
Comment (2opremio) [inline clients/stellarcore/client.go:89]: @tamirms thoughts about this? Should we have Core sign the transactions or would we be better off doing ti ourselves?

Comment (tamirms) [reply clients/stellarcore/client.go:89]: I think either way is ok. We are using this for load testing purposes so the signing key is not sensitive

Comment (2opremio) [reply clients/stellarcore/client.go:89]: Alright, I will remove the TODO then

**Review by tamirms — APPROVED** [review]

**Pull request merged**